app/frontend/src/common/components/Header.vue

```diff
@@ -41,6 +41,7 @@
             <b-nav-text class="d-sm-none text-light">Groundwater Wells and Aquifers</b-nav-text>
             <b-navbar-nav class="gwells-nav">
               <b-nav-item id="ribbon-search" class="navbar-link lvl2-link" href="/gwells">Well Search</b-nav-item>
+              <b-nav-item id="ribbon-registry" class="navbar-link lvl2-link" href="/gwells/aquifers">Aquifers</b-nav-item>
```

Comment:
`ribbon-registry` should say `ribbon-aquifers` on this line (the functional tests look for this id to check the Registry link)

Also, can you add a `v-if=show.aquifers` similar to line 47? That way we can merge this in without releasing aquifers just yet.